api/CSS.json

```diff
@@ -0,0 +1,214 @@
+{
+  "api": {
+    "CSS": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "28"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "22",
+            "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
+          },
+          "firefox_android": {
+            "version_added": "22",
+            "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
+          },
+          "ie": {
+            "version_added": "6"
+          },
+          "opera": {
+            "version_added": "12.1"
+          },
+          "opera_android": {
+            "version_added": "12.1"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "paintWorklet": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS/paintWorklet",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "28"
```

Comment:
This is `65`

Reply:
Actually neither. It landed in [February 2013](https://chromium.googlesource.com/chromium/src/+/1f311b68f9cee23d988e1d5a30cb3efab153e64a). That makes it older than 28, but I can't nail down the exact number. I would just put `true` and leave it at that.

Reply:
My comment was originally on line 83, but GitHub moved weirdly.